installation/kie-helm-charts.adoc

```diff
@@ -0,0 +1,409 @@
+= BAMOE CANVAS Helm Chart
+include::../partials/attributes.adoc[]
+
+Helm Charts is a technology that can enable you to install the multi-container services with a single command: Helm, https://helm.sh/[the package manager for kubernetes].
+
+Helm helps you manage Kubernetes applications and Helm Charts help you define, install, and upgrade even the most complex Kubernetes application.
+
+Charts are easy to create, version, share, and publish instead of using copy-and-paste.
+
+This chart can be used to deploy {CANVAS} image on a https://kubernetes.io[Kubernetes] cluster using the https://helm.sh[Helm package manager].
+
+== Additional requirements
+
+* Podman (for Linux)
+* Docker (for macOS)
+* Minikube
+
+== Components
+
+* {CANVAS}: Main application
+* Extended Services: Powers the DMN Runner and validator
+* CORS Proxy: Intended to solve CORS issues
+
+== Installing the Chart
+
+=== Default install
+
+Follow the command to install the chart with the release name `kie-sandbox`:
+
+[source, shell]
+----
+$ helm install kie-sandbox ./src
```

Comment:
The release name can be anything the user wants, maybe we should default to `bamoe-canvas` instead of `kie-sandbox`.